mobile/src/modules/home/components/HomeStackedAreaChart.tsx

```diff
@@ -99,9 +98,11 @@ export class HomeStackedAreaChart extends React.PureComponent<TProps, State> {
           if (convertBurstTimeToDate(transaction.timestamp) > d) {
             continue;
           }
-          if (priceInBTC) {
-            const priceInBTCOnThatDay = historicalPrice[atThatTime];
-            data[i][`${ACCOUNT_TOKEN}${accountIndex}`] = transaction.balance * priceInBTCOnThatDay.close;
+
+          if (showPricesIn === this.props.priceTypes[1] ||
```

Comment:
and here too, why not ` === PriceType.BTC` or just single condition `!== PriceType.BURST`?

Reply:
Forgot this one, thanks!